### ISSUE TITLE
connection refused with preview015

### ISSUE THREAD

**Issue by tomitrescak** [prisma/prisma2#844]
Hi, when I upgraded to preview015 following happened:

- First query executes fine
- Second query on I only get "connection refused" 👎 

```
Error:  
    Invalid `photon.()` invocation in ./src/bpmn_process_instance/tests/process_instance_resolver.test.js:23:58 
     
    Reason: { 
      "name": "RequestError", 
      "code": "ECONNREFUSED", 
      "host": "localhost:60721", 
      "hostname": "localhost", 
      "method": "POST", 
      "path": "/", 
      "protocol": "http:", 
      "url": "http://localhost:60721/", 
      "gotOptions": { 
        "path": "/", 
        "protocol": "http:", 
        "slashes": true, 
        "auth": null, 
        "host": "localhost:60721", 
        "port": "60721", 
        "hostname": "localhost", 
        "hash": null, 
        "search": null, 
        "query": null, 
        "pathname": "/", 
        "href": "http://localhost:60721/", 
        "retry": { 
          "methods": {}, 
          "statusCodes": {}, 
          "errorCodes": {} 
        }, 
        "headers": { 
          "user-agent": "got/9.6.0 (https://github.com/sindresorhus/got)", 
          "content-type": "application/json", 
          "accept": "application/json", 
          "accept-encoding": "gzip, deflate", 
          "content-length": 151 
        }, 
        "hooks": { 
          "beforeRequest": [], 
          "beforeRedirect": [], 
          "beforeRetry": [], 
          "afterResponse": [], 
          "beforeError": [], 
          "init": [] 
        }, 
        "decompress": true, 
        "throwHttpErrors": true, 
        "followRedirect": true, 
        "stream": false, 
        "form": false, 
        "json": true, 
        "cache": false, 
        "useElectronNet": false, 
        "body": "{\"query\":\"query {\\n  findManyUser {\\n    id\\n    email\\n    name\\n    description\\n    password\\n    privateKey\\n    publicKey\\n  }\\n}\",\"variables\":{}}", 
        "method": "POST", 
        "forceRefresh": true 
      } 
    } 
```

**Comment by tomitrescak** [comment]
Nevermind, it was a parallel query that brought prisma down :/
I did:

```
await Promise.all(processes.map(p => ctx.db.processes.findOne({ where: { id: p } })))
```

This crashed

**Comment by pantharshit00** [comment]
Hi,


You have closed this issue but this doesn't seem to be resolved. Can we make sure that this is resolved?

**Comment by tomitrescak** [comment]
@pantharshit00 not sure why my last message does not appear. Yes, it is resolved. The problem was that during those operations one of the constraints was failing. Unfortunately I was not getting any message about that, only the `connection refused` message. When I debugged the process, it was clearer. Thus, when I removed that constraint error, all started to work as expected.

Thanks for your care!
